packages/terser/test/test.js

```diff
@@ -15,6 +15,21 @@ test.serial('minify', async (t) => {
   t.falsy(output.map);
 });
 
+test.serial('minify with source map', async (t) => {
+  const bundle = await rollup({
+    input: 'test/fixtures/unminified.js',
+    plugins: [terser()]
+  });
+  const result = await bundle.generate({ format: 'cjs', sourcemap: true });
+  t.is(result.output.length, 2);
+  const [output] = result.output;
+
+  t.truthy(output.map);
+  t.is(output.map.version, 3);
+  t.is(output.map.file, 'unminified.js');
+  t.truthy(output.map.names);
```

Comment:
just caught this after clicking approve: an empty array will always be truthy. you want to check the length. a snapshot might be a good idea here.

Reply:
yep, you are completly right. While writing the test, the `deepEqual` operator did not occur to me, because `is` can not be used for array comparision.
I fixed the line.